.github/dependabot.yml

```diff
@@ -9,12 +9,10 @@ updates:
     # Pause Dependabot updates. Security updates are unaffected
     open-pull-requests-limit: 0
 
-    reviewers:
-      - "streamlit/core"
-
     labels:
-      - "needs triage"
-      - "infra"
+      - "security-assessment-completed"
```

Comment:
We probably shouldn't have this label applied by default since part of merging dependabot PRs generally involves sanity checking the changelog, and having to apply this label manually acts as a reminder to do so (plus I think by design this label should probably never be auto-applied).